src/modules/compliance/src/lib/procedures/FileRegexMatch.cpp

```diff
@@ -55,7 +56,11 @@ Result<Status> Match(const std::string& filename, const string& matchPattern, co
             mainMatched = true;
             if (stateRegex.HasValue())
             {
-                if (!regex_search(line, stateRegex.Value()))
+                assert(match.ready());
+                assert(match.size() > 0);
+                auto valueToMatch = match.size() > 1 ? match[1].str() : match[0].str();
+                OsConfigLogDebug(context.GetLogHandle(), "Value to match: %s", valueToMatch.c_str());
```

Comment:
>Value t [](http://example.com/codeflow?start=58&length=7)

May be useful to add the function/method name. To keep things simple, we do not automatically capture that (as context we capture the source file name and line number). In a case like this logging like this can be clear for us, but not for a customer that may read our logs.

Reply:
This comment now applies to line 67 (code changed from previous commit)